docs/versions/latest/Release.md

```diff
@@ -99,6 +99,26 @@ When using SSH with typescript, the `exitCode` method has the type `void`. But t
 #### Current Behaviour
 Method `.exitCode` should return type `Number` instead of type `void`
 
+### Upgrade VCD archetype to support Angular 15
+VMware Cloud Director v10.6 is going to drop support for Angular v9 or less
```

Comment:
@vganevjb  What is the minimum VCD version to support Angular 15?

Reply:
I successfully tested on 10.4.0, 10.4.1, 10.4.2 and 10.5.0